### PR TITLE
dark theme support

### PR DESCRIPTION
**Pull request by ostreifel**
image

Test work item
https://ottost.visualstudio.com/Sample%20Project/_workitems/edit/122500

### CODE REVIEW

**Comment by ostreifel** [comment]
See #75

**Comment by nschonni** [comment]
I've noticed these pulls on some other things, does this mean IE11 is no longer supported?

File: src/multi-selection.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#container {
+    padding-bottom: 3px;
+    padding-right: 3px;
```
Comment (ostreifel) [inline src/multi-selection.scss]: Fixes the clipping on the rhs of the control

**Comment by ostreifel** [comment]
@nschonni Nice catch, update to use mixin workaround for ie

File: src/multi-selection.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
+// ie does not support variables, use mixins as workaround
+@mixin border-color($varName, $defaultR, $defaultG, $defaultB) {
+    border-color: rgb($defaultR, $defaultG, $defaultB);
+    border-color: rgb(var($varName, defaultR, $defaultG, $defaultB));
```
Comment (nschonni) [inline src/multi-selection.scss]: Missed the `$` for the Sass variable

**Pull request merged**